GVRf/Framework/src/org/gearvrf/GVRViewManager.java

```diff
@@ -104,13 +104,16 @@
     private GVRScreenshotCallback mScreenshotCenterCallback = null;
     private GVRScreenshotCallback mScreenshotLeftCallback = null;
     private GVRScreenshotCallback mScreenshotRightCallback = null;
+    private GVRScreenshot3DCallback mScreenshot3DCallback = null;
+    ByteBuffer readbackBuffer = null;
+    int readbackBufferWidth = 0, readbackBufferHeight = 0;
```

Comment:
These two fields appear to only be referenced  in `readRenderResult()` - they should probably be local variables.